_includes/async/keyboard-download.php

```diff
@@ -1,220 +0,0 @@
-<?php
-  define('DEBUG', 0); // 0 = off, 1 = lots of logging but don't force a build, 2 = force a build
-
-  if(!is_cli()) {
-    echo 'keyboard-download.php must be run from the command line, not web server environment.';
-    exit(1);
-  }
-
-  require_once(dirname(__FILE__)."\\..\\includes\\servervars.php");
-
-  if($argv < 5) {
-    error_log('Parameters: cid id platform mode bearer_token(teamcity)');
-    exit(1);
-  }
-
-  $cid = $argv[1];
-  $id = $argv[2];
-  $platform = $argv[3];
-  $mode = $argv[4];
-  $bearer_token = $argv[5];
-
-  if(DEBUG) {
-    error_log("Input parameters: cid=$cid id=$id platform=$platform mode=$mode bearer_token=$bearer_token");
-  }
-
-  function fail($s) {
-    error_log('FAIL: '.$s);
-    exit(1);
-  }
-
-  $downloads = getKeyboardDownloadData($id);
-
-  recordGoogleAnalyticsEvent($cid, $id, $platform, $mode);
```

Comment:
I don't see where any Google analytics references are maintained under this PR's 'additions'; are we intentionally dropping use of Google Analytics for keyboard downloads?  Seems odd not to note that explicitly on the PR overview if so.

Reply:
That's a good catch. While the install.php page will capture keyboard downloads in Analytics, we would lose the events which we are going to be using for download statistics. I will add something into install.php (front-end to avoid missing stats through caching) to replicate this.